ui/src/components/NodeListTable.js

```diff
@@ -391,12 +390,14 @@ const NodeListTable = (props) => {
       location.pathname.endsWith('pods');
 
     if (isTabSelected) {
-      // When switch between the nodes, keep the same tab selected
-      const currentTab = location?.pathname?.split('/')?.pop();
-      history.push(`/newNodes/${nodeName}/${currentTab}`);
+      const newPath = location.pathname.replace(
+        // eslint-disable-next-line no-useless-escape
```

Comment:
It's actually an [eslint-error](https://stackoverflow.com/a/47277197) because of the regex: Unnecessary escape character: \/ no-useless-escape 
But I think the regex is correct. That's why I disable it, maybe the message is not easy to understand.

Reply:
As mentioned in the answer you linked, you don't need to escape the forward slash inside the character class e.g. you could write `/\/newNodes\/[^/]*\//` and have the same behavior:

```
> var nodeName = "foo";
undefined
> var str = "/newNodes/BAR/";
undefined
> str.replace(/\/newNodes\/[^\/]*\//, `/newNodes/${nodeName}/`); 
'/newNodes/foo/'
> str.replace(/\/newNodes\/[^/]*\//, `/newNodes/${nodeName}/`); 
'/newNodes/foo/'
```

Did you try that version?
Is ESLint happy with this regex? Or do you still have the warning?

Reply:
I confirm `/\/newNodes\/[^/]*\//` is working correctly!